src/lib/date.ts

```diff
@@ -27,7 +80,81 @@ export function formattedOpeningHours(startAt, endAt) {
   }
 }
 
+export function singleHour(date) {
+  // 5pm
+  // 5:30pm
+  let hour
+  const singleHour = moment(date)
+  if (singleHour.minutes() === 0) {
+    hour = singleHour.format("ha")
+  } else {
+    hour = singleHour.format("h:mma")
+  }
+  return hour
+}
+
+export function singleDate(date) {
+  // Aug 5, 5:00pm
```

Comment:
Same here -- its idomatic to use DocBlocks annotated at the top of the function, e.g., 
```
/**
 * 
 *
 */
```